packages/dropdown-react/src/Dropdown.tsx

```diff
@@ -40,6 +45,10 @@ function focusSelected(listEl: HTMLElement, listId: string, selected: string | u
     focusedItem && focusedItem.focus();
 }
 
+export function makeSelectValueFrom(item: string | SelectValue) {
```

Comment:
In my mind `make` is a keyword for a function that returns a function. I'd prefer `getSelectValue()` here

Reply:
And since both uses this, i'd prefer it to be split out to it's own file or utility file

Reply:
And write a small litte test for it :dancer: